cfme/fixtures/v2v_fixtures.py

```diff
@@ -0,0 +1,449 @@
+import itertools
+import json
+from collections import namedtuple
+
+import fauxfactory
+import pytest
+from riggerlib import recursive_update
+from widgetastic.utils import partial_match
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import setup_or_skip
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.v2v.infrastructure_mapping import InfrastructureMapping as InfraMapping
+
+
+FormDataVmObj = namedtuple("FormDataVmObj", ["infra_mapping_data", "vm_list"])
+
+
+@pytest.fixture(scope="module")
+def v2v_provider_setup(request, source_provider, provider):
+    """ Fixture to setup providers """
+    V2vProviders = namedtuple("V2vProviders", ["vmware_provider", "rhv_provider", "osp_provider"])
+    vmware_provider, rhv_provider = None, None
+    for provider in [source_provider, provider]:
```

Comment:
I think its a bad practice to overload `provider` here, you need a separate variable name for the loop iterations